src/pyodide/pyodide-bootstrap-helper.js

```diff
@@ -50,39 +48,25 @@ function initializePackageIndex(pyodide, lockfile) {
 // These packages are currently embedded inside workerd and so don't need to
 // be separately installed.
 const EMBEDDED_PYTHON_PACKAGES = [
-  "tqdm",
-  "openai",
```

Comment:
I'm not sure about other packages but this one is critical. Did you remove these packages from the tarball? If so the langchain test won't work.

Reply:
Yeah since langchain doesn't have a direct requirement for openai `pip install langchain fastapi` misses `openai`. So I guess you need `pip install langchain openai fastapi` to get all of these.